pkgs/by-name/mk/mkwebfont/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  mkwebfont,
+  rustPlatform,
+  testers,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
More preferred over rec whenever possible. Also moves the recursion from `callPackage` to the builder